tests/e2e/resources/installation_config/cognito-rds-s3.yaml

```diff
@@ -373,8 +373,8 @@ aws-secrets-manager:
     pods:
       namespace: kubeflow
       labels:
-        key: app
-        value: aws-secrets-sync
+        - key: app
+          value: aws-secrets-sync
 
 ack-sagemaker-controller:
```

Comment:
more issues with this config, sagemaker-controller and aws-telemetry does not have updated installation config schema